tools/actions/commit-queue.sh

```diff
@@ -37,6 +37,18 @@ gitHubCurl() {
        --header 'content-type: application/json' "$@"
 }
 
+commit_queue_failed() {
+  gitHubCurl "$(labelsUrl "${1}")" POST --data '{"labels": ["'"${COMMIT_QUEUE_FAILED_LABEL}"'"]}'
+
+  # shellcheck disable=SC2154
+  cqurl="${GITHUB_SERVER_URL}/${OWNER}/${REPOSITORY}/actions/runs/${GITHUB_RUN_ID}"
+  jq -n --arg content "<details><summary>Commit Queue failed</summary><pre>$(cat output)</pre><a href='$cqurl'>$cqurl</a></details>" '{body: $content}' > output.json
+  cat output.json
+
+  gitHubCurl "$(commentsUrl "${1}")" POST --data @output.json
+
+  rm output output.json
+}
 
 # TODO(mmarchini): should this be set with whoever added the label for each PR?
 git config --local user.email "github-bot@iojs.org"
```

Comment:
Still iojs.org?

Reply:
Out of curiosity, why?  Just haven't gotten around to changing it?

Reply:
Probably. I don't have context on it tbh.